core/src/cluster_info.rs

```diff
@@ -114,6 +114,11 @@ pub const DEFAULT_CONTACT_DEBUG_INTERVAL_MILLIS: u64 = 10_000;
 pub const DEFAULT_CONTACT_SAVE_INTERVAL_MILLIS: u64 = 60_000;
 /// Minimum serialized size of a Protocol::PullResponse packet.
 const PULL_RESPONSE_MIN_SERIALIZED_SIZE: usize = 161;
+/// Minimum stake that a node should have so that its CRDS values are
+/// propagated through gossip (ContactInfo is exempted).
```

Comment:
Can you please clarify units here.  I initially assumed ◎1 but I think it's 1 lamport

Reply:
Honestly I am not confident what the unit is. There is `stake: u64` throughout the code:
https://github.com/solana-labs/solana/search?p=1&q=%22stake%3A+u64%22
without a unit. The code here is only a consumer of the stakes hashmap. 
Should I assume it is in lamports?

Reply:
actually sorry, this is fine.  Lamports are always assumed in the code base

Reply:
Then I guess this should be `LAMPORTS_PER_SOL`, instead of `1`, right?

Reply:
If we want ◎1 to be the minimum, yes.  1 lamport seemed just as good though, I think the point is mostly that it's not zero righit?

Reply:
I got the units wrong, but my initial intention was to set this to 1 sol. My suggestion is to set this to at least `LAMPORTS_PER_SOL`. 1 lamport seems too low to work around this defence. What would you suggest?

Reply:
As a UX convenience, can this be `SOME_NICE_ROUND_NUMBER - STAKE_RENT_EXEMPT_MINIMUM`?

Reply:
@t-nelson I thought `STAKE_RENT_EXEMPT_MINIMUM` is some constant in the code; but I cannot find any such value. Would you mind clarify?

@mvines any objection to increasing this value to `LAMPORTS_PER_SOL`?

Reply:
> any objection to increasing this value to LAMPORTS_PER_SOL?

No objections

Reply:
> @t-nelson I thought `STAKE_RENT_EXEMPT_MINIMUM` is some constant in the code; but I cannot find any such value. Would you mind clarify?

Sorry, no that's just pseudo code.  You can calculate it with, https://github.com/solana-labs/solana/blob/1061d021c97a604672fc739e307b0d572ad98ce9/programs/stake/src/stake_state.rs#L74-L76

Reply:
◎1 seems close enough, I don't think we need to get fancy with rent in this computation.